plugins/modules/zos_backup_restore.py

```diff
@@ -351,6 +349,28 @@
     sms_storage_class: DB2SMS10
     sms_management_class: DB2SMS10
 """
+RETURN = r"""
+changed:
+  description:
+    - Indicates if the operation made changes.
+    - C(true) when backup/restore was successful, C(false) otherwise.
+  returned: always
+  type: bool
+  sample: true
+backup_name:
+  description:
+    - The backup file or data set name that was processed.
+    - Matches the I(backup_name) parameter provided as input.
+  returned: always
+  type: str
+  sample: "/u/oeusr03/my_backup.dzp"
+message:
+  description:
+    - Empty ("") for successfull and failure operations.
```

Comment:
Even though this is true at the moment, we don't want to tell users "oh by the way this does nothing". I think is better to say:
```suggestion
    - Returns any important messages about the modules execution, if any.
```

At this point it might be empty, but could be used for something in the future. The truth is it won't be, this looks like an attempt to define `msg` this module is old and original developer may have made a mistake. This is something we can remove in the future releases as a deprecation.

But at this point lets use this phrase instead.

Reply:
Changes have been added .